doc/compressed_transactions.md

```diff
@@ -0,0 +1,49 @@
+| CV->LockTime                   | 2 Bits    | 0x00 if the LockTime is zero, 0x01 if the LockTime has been shortened to the sixteen least significant bits, 0x02 if is not been shortened and signals the existence of a future VarInt for the LockTime. |
+| CV->Coinbase                   | 1 Bits    | Boolean to determine if this transaction is a Coinbase Transaction. |
+| Version VarInt                 | 1-4 Bytes | If the Version was greater then 3 then this is an Optional Variable Integer containing the Version. |
+| Input Count VarInt             | 1-4 Bytes | If the Input Count was greater then 3 then this is an Optional Variable Integer containing the Input Count. |
+| Output Count VarInt            | 1-4 Bytes | If the Output Count was greater then 3 then this is an Optional Variable Integer containing the Output Count. |
+| LockTime VarInt                | 1-4 Bytes | An Optional Variable Integer containing either the full LockTime or the shortened sixteen least significant bits. |
+| Input/Output Stream(IO/Stream) | ? Bytes   | A Variable Length Bit Stream padded to full Bytes, Contains metadata on the Inputs and Outputs of the Transaction. |
+| IO/Stream->Signature Signed    | 1 Bits    | Boolean to determine if this Input has been Signed. |
+| IO/Stream->Signature Compressed| 1 Bits    | Boolean to determine if this Input's Signature has been Compressed. |
+| IO/Stream->Txid Compressed     | 1 Bits    | Boolean to determine if this Input's Txid has been Compressed. |
+| IO/Stream->Standard Hash       | 1 Bits    | Boolean to determine if this Input's Signature Hash Type is (0x00 or 0x01) or something else. |
+| IO/Stream->Hash Type           | 1 Bits    | Boolean to determine if this Input's Signature Hash Type is 0x00 or 0x01. |
+| IO/Stream->Standard Sequence   | 1 Bits    | Boolean to determine if this Input's Sequence is 0x00000000, 0xFFFFFFF0, 0xFFFFFFFE or 0xFFFFFFFF. |
+| IO/Stream->Sequence Encoding   | 2 Bits    | Encoded Sequence to match the set of Standard Sequences. |
+| IO/Stream->Encoded Script Type | 3 Bits    | Encoded Script Type. |
+| Input(I)                       | ? Bytes   | For each Input Encode the following. |
+| I->Sequence                    | 1-4 Bytes | If the Sequence was non standard then this contains a Variable Integer containing the Sequence. |
+| I->Txid Block Height           | 1-4 Bytes | If the Txid was Compressed then this contains a Variable Integer containing the Block Height of the Txid. |
+| I->Txid Block Index            | 1-4 Bytes | If the Txid was Compressed then this contains a Variable Integer containing the Block Index of the Txid. |
+| I->Txid                        | 32 Bytes  | If the Txid was Uncompressed then this is a Stream of 32 Bytes containing the Txid. |
+| I->Vout                        | 1-4 Bytes | A Variable integer containing the Vout. |
```

Comment:
In 61504d1a4d7ea545b1880686e426b1acae7d26f0:

Let's make the block index cover the vout, and then we only need the vout for uncompressed transactions (indicated by block height/index 0/0).

Reply:
Fixed, Thanks!